docs/software/system_stats.md

```diff
@@ -635,83 +635,94 @@ Homer is a modern and lightweight dashboard & homepage for your services
 
 === "Theming - Dracula"
 
-    To apply the [Dracula theme](https://draculatheme.com/static/img/screenshots/homer.png) to Homer follow the steps below:
+    To apply the [Dracula theme](https://draculatheme.com/static/img/screenshots/homer.png) to Homer execute these steps:
 
-    ```sh
-    apt install git
-    git clone https://github.com/dracula/homer.git
-    cp homer/custom.css /var/www/homer/assets/custom.css
-    cp homer/dracula-background.png /var/www/homer/assets/dracula-background
-    ```
+    1. Get theme:
+
+        ```sh
+        curl -fL 'https://raw.githubusercontent.com/dracula/homer/master/custom.css' -o /var/www/homer/assets/custom.css
+        curl -fL 'https://raw.githubusercontent.com/dracula/homer/master/dracula-background' -o /var/www/homer/assets/dracula-background
+        ```
 
-    Edit your `config.yml` with the below command:
+    1. Edit your `config.yml` with:
 
-    ```sh
-    nano /var/www/homer/assets/config.yml
-    ```
+        ```sh
+        nano /var/www/homer/assets/config.yml
+        ```
 
-    And add the following lines:
+    1. Add the following lines:
 
-    ```yaml
-    # Will load Dracula theme
-    stylesheet:
-      - "assets/custom.css"
-    ```
+        ```yaml
+        # Will load Dracula theme
+        stylesheet:
+        - "assets/custom.css"
+        ```
 
 === "Theming - macOS"
 
-    To apply the [macOS styled theme](https://raw.githubusercontent.com/WalkxCode/Homer-Theme/main/preview.png) to Homer follow the steps below:
+    To apply the [macOS styled theme](https://raw.githubusercontent.com/WalkxCode/Homer-Theme/main/preview.png) to Homer execute these steps:
 
-    ```sh
-    apt install git
-    git clone https://github.com/WalkxCode/Homer-Theme.git
-    cp Homer-Theme/custom.css /var/www/homer/assets/custom.css
-    cp Homer-Theme/wallpaper.jpeg /var/www/homer/assets/wallpaper.jpeg
-    cp -r Homer-Theme/fonts /var/www/homer/assets/
-    ```
+    1. Get theme:
+
+        ```sh
+        cd /tmp
+        curl -fLO 'https://github.com/walkxcode/homer-theme/archive/main.tar.gz'
+        tar xf main.tar.gz
+        rm main.tar.gz
+        cp Homer-Theme-main/custom.css /var/www/homer/assets/custom.css
+        cp Homer-Theme-main/wallpaper.jpeg /var/www/homer/assets/wallpaper.jpeg
```

Comment:
Hmm, actually IMO it's better to not overwrite the original wallpaper but store it as dedicated file, like `wallpaper-macos.jpeg` and adjust the `background-image` in `config.yml` accordingly, isn't it? That way also no manual recovery step is required after updating.

Reply:
Ah, there is not default wallpaper, then all is fine.